azuredeploy.json

```diff
@@ -289,7 +292,8 @@
         "mssqlDbEdition": {
             "allowedValues": [
                 "Basic",
-                "Standard"
```

Comment:
See an issue with offering Premium across all t-shirt sizes here. A customer may deploy a smaller t-shirt size and end up being surprised at the incremental cost with Premium SKUs in general.